### ISSUE TITLE
The scattering file forces random seed to be set to 0

### ISSUE THREAD

**Issue by ilias-audio** [gdalsanto/flamo#19]
https://github.com/gdalsanto/flamo/blob/44978617eca766fd2b5093ad2fbc07d139a1e200/flamo/auxiliary/scattering.py#L7

This causes the seed to be always set to 0. 

Quick test to check this error:

`import torch`
`from flamo.processor import dsp`
`print(f"Initial Seed: {torch.initial_seed()}")`

It will return zero. 

Commenting those lines "solves" the problem, but I don't know if it affects other things...
